### PR TITLE
Add more words

### PR DESCRIPTION
**Pull request by sebweb3r**
.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -16437,6 +16441,8 @@ liveing->living
 livel->level
 livetime->lifetime
 livley->lively
+lizens->license
```
[the remark this answers is not in the document]
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16444]: Yes it is. I have a colleague, who makes this mistake regularly. Grep.app only finds some entries for lizense. 
I can add that too, or move both to a local copy of the dictionary.

[the remark this answers is not in the document]
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16444]: lizensing is already in btw.

**Review by ssbarnea — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:16444]: I think put it in for now and we can review. Can you add lizense too.

I was trying to add codespell to another codebase and it keeps getting tripped up on Norwegian I think where there are comments in that language and the words are in our dictionary as typos.

[the remark this answers is not in the document]
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:16444]: added.

**Review by peternewman — APPROVED** [review]

**Pull request merged**